Infrastructure.md

```diff
@@ -0,0 +1,58 @@
+# Infrastructure Setup
+
+The provisioning system depends on a central master, hosted at http://classroom.puppet.com
```

Comment:
yeah that makes sense. Maybe we could add instructions to a troubleshooting page for a standalone build with puppet apply.